helpers/helpers.v1.d/go

```diff
@@ -18,41 +18,32 @@
 # along with this program. If not, see <http://www.gnu.org/licenses/>.
 #
 
-ynh_go_try_bash_extension() {
-    if [ -x src/configure ]; then
-        src/configure && make -C src || {
-            ynh_print_info --message="Optional bash extension failed to build, but things will still work normally."
-        }
-    fi
-}
-
 goenv_install_dir="/opt/goenv"
 go_version_path="$goenv_install_dir/versions"
-# goenv_ROOT is the directory of goenv, it needs to be loaded as a environment variable.
+
+# GOENV_ROOT is the directory of goenv, it needs to be loaded as a environment variable.
 export GOENV_ROOT="$goenv_install_dir"
+export goenv_root="$goenv_install_dir"
```

Comment:
(are we sure that having that same var in lowercase is actually useful for anything x_x ... i guess it comes from copypasting the same stuff as for ruby but that sounds very meh)

Reply:
I don't think, it's usefull, but was in ruby herlper